src/sentry/models/group.py

```diff
@@ -155,14 +174,24 @@ def filter_by_event_id(self, project_ids, event_id):
         # see above for explanation as to why we're
         # looking at both Event and EventMapping
         for model in Event, EventMapping:
+            event_id_lower = event_id.lower()
             group_ids.update(
                 model.objects.filter(
                     project_id__in=project_ids,
-                    event_id=event_id,
+                    event_id=event_id_lower,
```

Comment:
you could just do event_id__in here to simplify this logic (i think)

Reply:
Yeah, I'd do `event_id__in={event_id, event_id_lower}`.

I verified, and it doesn't mess with the query planner.

I _would_ be curious if this yields more than 1 row though since it's technically possible.

Reply:
@mattrobenolt we have been writing only lowercase on the ingest side for at least 90d, I think it should be reasonably safe to assume that everything in the database is lowercase. Although IDK, maybe there are some exceptions for models that don't get cleaned up the same way (EventMapping?)

Reply:
Ok, then if we're comfortable knowing that no events have uppercase, then we can ignore all of this. EventMapping is a significantly smaller table is is also aggressively trimmed to 90d.